topics/admin/tutorials/interactive-tools/tutorial.md

```diff
@@ -249,6 +252,9 @@ The GIE Proxy is written in [Node.js][nodejs] and requires some configuration. T
 >    | `gie_proxy_virtualenv`         | path (string) | Path of virtualenv into which nodeenv/Node.js/npm will be installed   |
 >    | `gie_proxy_setup_service`      | string        | Whether to configure the proxy as a service, only option is `systemd` |
 >    | `gie_proxy_sessions_path`      | path (string) | Path of Interactive Tools sessions map                                |
+>    |--------------------------------|---------------|-----------------------------------------------------------------------|
```

Comment:
```suggestion
```

is there a reason to add the lines here?

Reply:
No :rofl:, removed 96d43ab1f7311cf32f3ddf2df2098e3502309d5e